### PR TITLE
[DOCS] Added OSS repo info.

### PR DESCRIPTION
**Pull request by debadair**

### CODE REVIEW

File: docs/reference/setup/install/rpm.asciidoc [hunk 1]
```diff
@@ -90,6 +90,9 @@ sudo zypper install elasticsearch <3>
 
 endif::[]
 
+The OSS only distribution is available from the
```
Comment (jasontedor) [inline docs/reference/setup/install/rpm.asciidoc]: Nit: `OSS-only`?

File: docs/reference/setup/install/rpm.asciidoc [hunk 2]
```diff
@@ -112,10 +115,9 @@ sudo rpm --install elasticsearch-{version}.rpm
 --------------------------------------------
 <1> Compares the SHA of the downloaded RPM and the published checksum, which should output
     `elasticsearch-{version}.rpm: OK`.
-    
-Alternatively, you can download the following package, which contains only 
-features that are available under the Apache 2.0 license: 
-https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{version}.rpm    
+
+The OSS only distribution is available at
```
Comment (jasontedor) [inline docs/reference/setup/install/rpm.asciidoc]: Nit: `OSS-only`?

**Review by jasontedor — CHANGES_REQUESTED** [review]
I think we need a similar change on the Debian page?

**Comment by colings86** [comment]
@debadair Could we add the information to the deb package instructions and get this merged?

**Pull request closed without merging**